### PR TITLE
Team 5 Work Areas

### PR DESCRIPTION
**Pull request by JamesTev**
Roles of T5 members as suggested by @NGovani

### CODE REVIEW

**Review by yp717 — APPROVED** [review]
lgtm

**Review by ezgimez — CHANGES_REQUESTED** [review]
delete the repetition table, s'il vous plait

File: 20_appendix_roles/teams/team5.tex [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+    \textbf{Team Member} & \textbf{Areas}     \\ \hline
+    Catherine Chaya  & Design, Report, Sim         \\
+    Philippos Arkis Hadjimarkou  & Agent, Report, Sim                 \\
+    Justin Ma     & Design, Report, Agent, Sim        \\
+    Semina Mertzani     & Design, Report, Sim \\
+    Anh Nguyen    & Report, Agent, Sim                      \\
+    James Teversham  & Infra, Design, Report, Agent                        \\
+    \hline
+\end{tabular}
+\caption{Areas of work that Team 5 members have contributed to}
```
Comment (ezgimez) [inline 20_appendix_roles/teams/team5.tex:17]:
```suggestion
\caption{Areas of work that Team 5 members have contributed to.}
```

**Review by ezgimez — APPROVED** [review]
let's mergeeeeeeee

**Pull request merged**